app-authz-jakarta-servlet/config/realm-import.json

```diff
@@ -82,11 +80,11 @@
     {
       "clientId": "authz-servlet",
       "enabled": true,
-      "baseUrl": "http://localhost:8080/authz-servlet",
-      "adminUrl": "http://localhost:8080/authz-servlet",
+      "baseUrl": "http://127.0.0.1:8080/authz-servlet",
```

Comment:
Any specific reason why all those URLs need to be changed from `localhost` to `127.0.01` ?

Reply:
Hope is fine now.